modules/openxr/doc_classes/OpenXRExtensionWrapperExtension.xml

```diff
@@ -90,6 +90,21 @@
 				Called right after the main swapchains are (re)created.
 			</description>
 		</method>
+		<method name="_on_post_draw_viewport" qualifiers="virtual">
+			<return type="void" />
+			<param index="0" name="viewport" type="RID" />
+			<description>
+				Called right after the given viewport is rendered.
+				[b]Note:[/b] The draw commands may only be queued at this point, not executed.
```

Comment:
```suggestion
				[b]Note:[/b] The draw commands should only be queued at this point, not executed.
```
Clearer IMO, if the meaning was "only queue them, don't execute them", otherwise if the meaning is "they might or might not be executed, but will be queued" I'd use:
```suggestion
				[b]Note:[/b] The draw commands might only be queued at this point, not executed.
```
"May" is a sneaky word

Reply:
Thanks! I've updated it to your 2nd suggestion in my latest push